tools/dm: refine dashboard description (#1243)
pingcap/docs-cn: tools/dm/monitor.md

## Task

> Grafana dashboard 默认名称为 `DM`。 

Reviewer: If you use "Grafana dashboard 默认名称为 `DM`", users might think it means "the default name of Grafana dashboard is `DM`".

### overview

> > overview 下包含运行当前选定 task 的所有 DM-worker instances 的部分监控指标。当前默认告警规则只针对于单个 DM-worker instance。

Reviewer: ```suggestion
overview 下包含运行当前选定 task 的所有 DM-worker instance 的部分监控指标。当前默认告警规则只针对于单个 DM-worker instance。
```

| metric 名称 | 说明 | 告警说明 |
|:----|:------------|:----|
| task state | 同步子任务的状态 | N/A |
| storage capacity | relay log 占有的磁盘的总容量  | N/A |
| storage remain | relay log 占有的磁盘的剩余可用容量  | N/A |
| binlog file gap between master and relay | relay 与上游 master 相比落后的 binlog file 个数 | N/A |

> | load progress | loader 导入过程的进度百分比，值变化范围为：0 %- 100 %  | N/A |

Reviewer: ```suggestion
| load progress | loader 导入过程的进度百分比，值变化范围为：0% - 100%  | N/A |
```

> | binlog file gap between master and syncer | 与上游 master 相比落后的 binlog file 个数。| N/A |

Reviewer: ```suggestion
| binlog file gap between master and syncer | 与上游 master 相比落后的 binlog file 个数 | N/A |
```

## Task

> 在 Grafana dashboard 中，DM 默认名称为 `DM`。 

Reviewer 2: is `DM-task` better?

Author: sounds good, then change the dashboard in DM-Ansible?

Author: changed

### overview

overview 下包含运行当前选定 task 的所有 DM-worker instance 的部分监控指标。当前默认告警规则只针对于单个 DM-worker instance。

| metric 名称 | 说明 | 告警说明 |
|:----|:------------|:----|
| task state | 同步子任务的状态 | N/A |
| storage capacity | relay log 占有的磁盘的总容量  | N/A |
| storage remain | relay log 占有的磁盘的剩余可用容量  | N/A |
| binlog file gap between master and relay | relay 与上游 master 相比落后的 binlog file 个数 | N/A |
| load progress | loader 导入过程的进度百分比，值变化范围为：0% - 100%  | N/A |

> | binlog file gap between master and syncer | 与上游 master 相比落后的 binlog file 个数 | N/A |

Reviewer 2: ```suggestion
| binlog file gap between master and syncer | 与上游 master 相比 binlog replication 落后的 binlog file 个数 | N/A |
```

|:----|:------------|:----|
| storage capacity | relay log 占有的磁盘的总容量  | N/A |
| storage remain | relay log 占有的磁盘的剩余可用容量  | 小于 10G 的时候需要告警 |
| process exits with error | relay log 在 DM-worker 内部遇到错误并且退出了  | 立即告警 |
| relay log data corruption | relay log 文件损坏的个数 | 立即告警 |
| fail to read binlog from master | relay 从上游的 MySQL 读取 binlog 时遇到的错误数 | 立即告警 |
| fail to write relay log | relay 写 binlog 到磁盘时遇到的错误数 | 立即告警 |
| binlog file index | relay log 最大的文件序列号。如 value = 1 表示 relay-log.000001 | N/A |
| binlog file gap between master and relay | relay 与上游 master 相比落后的 binlog file 个数 | 落后 binlog file 个数超过 1 个（不含 1 个）且持续 10 分钟时 |
| binlog pos | relay log 最新文件的写入 offset  | N/A |
| read binlog duration | relay log 从上游的 MySQL 读取 binlog 的时延，单位：秒 |  N/A |
| write relay log duration | relay log 每次写 binlog 到磁盘的时延，单位：秒| N/A |
| binlog size | relay log 写到磁盘的单条 binlog 的大小 | N/A |

### Dumper

下面 metrics 仅在 `task-mode` 为 `full` 或者 `all` 模式下会有值。

| metric 名称 | 说明 | 告警说明 |
|:----|:------------|:----|
| dump process exits with error | dumper 在 DM-worker 内部遇到错误并且退出了 | 立即告警 |

### Loader

下面 metrics 仅在 `task-mode` 为 `full` 或者 `all` 模式下会有值。

| metric 名称 | 说明 | 告警说明 |
|:----|:------------|:----|

> | load progress | loader 导入过程的进度百分比，值变化范围为：0%- 100%  | N/A |

Reviewer: ```suggestion
| load progress | loader 导入过程的进度百分比，值变化范围为：0% - 100%  | N/A |
```

> | write relay log duration | relay log 每次写 binlog 到磁盘的时延，单位：秒。| N/A |

Reviewer: Please help remove this period. Thanks.

Author: removed, thanks.

| execution latency | syncer 执行 transaction 到下游的耗时，单位：秒 | N/A |
| unsynced tables | 当前子任务内还未收到 shard DDL 的分表数量 | N/A |
| shard lock resolving | 当前子任务是否正在等待 shard DDL 同步，大于 0 表示正在等待同步 | N/A |

> ## Instances

Reviewer: ```suggestion
## Instance
```

Reviewer: or "instance"?

> Grafana dashboard 默认名称为 `DM-instances`。

Reviewer: ```suggestion
在 Grafana dashboard 中，instance 的默认名称为 `DM-instances`。
```

> > 当前默认告警规则在 [Task](#task) 部分已有说明，此处不再重复。

Reviewer: ```suggestion
当前默认告警规则参见 [Task](#task) 部分。
```

> 当前默认告警规则参见 [Task](#task) 部分。

Reviewer 2: 这个表述少了点什么吗？

Author: add more descriptions.

### Relay log

| metric 名称 | 说明 | 告警说明 |
|:----|:------------|:----|
| storage capacity | relay log 占有的磁盘的总容量  | N/A |
| storage remain | relay log 占有的磁盘的剩余可用容量  | N/A |
| process exits with error | relay log 在 DM-worker 内部遇到错误并且退出了  | N/A |
| relay log data corruption | relay log 文件损坏的个数 | N/A |

> | fail to read binlog from master | relay 从上游的 mysql 读取 binlog 时遇到的错误数 | N/A |

Reviewer: ```suggestion
| fail to read binlog from master | relay 从上游的 MySQL 读取 binlog 时遇到的错误数 | N/A |
```

| fail to write relay log | relay 写 binlog 到磁盘时遇到的错误数 | N/A |
| binlog file index | relay log 最大的文件序列号。如 value = 1 表示 relay-log.000001 | N/A |
| binlog file gap between master and relay | relay 与上游 master 相比落后的 binlog file 个数 | N/A |
| binlog pos | relay log 最新文件的写入 offset  | N/A |
| read binlog duration | relay log 从上游的 MySQL 读取 binlog 的时延，单位：秒 |  N/A |

> | write relay log duration | relay log 每次写 binlog 到磁盘的时延，单位：秒。| N/A |

Reviewer: ```suggestion
| write relay log duration | relay log 每次写 binlog 到磁盘的时延，单位：秒 | N/A |
```

| binlog size | relay log 写到磁盘的单条 binlog 的大小 | N/A |

### task

| metric 名称 | 说明 | 告警说明 |
|:----|:------------|:----|
| task state | 同步子任务的状态 | N/A |

> | load progress | loader 导入过程的进度百分比，值变化范围为：0 %- 100 %  | N/A |

Reviewer: ```suggestion
| load progress | loader 导入过程的进度百分比，值变化范围为：0% - 100%  | N/A |
```

> | binlog file gap between master and syncer | 与上游 master 相比落后的 binlog file 个数。| N/A |

Reviewer: ```suggestion
| binlog file gap between master and syncer | 与上游 master 相比落后的 binlog file 个数 | N/A |
```

> | binlog file gap between master and syncer | 与上游 master 相比落后的 binlog file 个数 | N/A |

Reviewer 2: ditto with L28

Author: addressed.
